backend/app/users/sensitive_personal_data/types.py

```diff
@@ -0,0 +1,47 @@
+"""
+This module contains the types used for storing sensitive personal data.
+"""
+
+from enum import Enum
+
+from pydantic import BaseModel, Field
+
+
+class SensitivePersonalDataStatus(Enum):
+    """
+    Represents the status of the sensitive personal data
+    """
+
+    NOT_REQUIRED = "NOT_REQUIRED"
+    REQUIRED = "REQUIRED"
+    CREATED = "CREATED"
+
+
+class CreateSensitivePersonalDataRequest(BaseModel):
+    """
+    Represents the request body for creating sensitive personal data.
+    """
+
+    rsa_key_id: str = Field(description="The key ID of the RSA key used to encrypt the AES key", examples=["key_123"])
+    """
+    The key ID of the RSA key used to encrypt the AES key.
+    """
+
+    aes_encryption_key: str = Field(description="RSA encrypted AES key used to encrypt/decrypt the sensitive user data",
```

Comment:
max set to 1000, the logic is: aes key is 256bits,  when encrypted with RSA 4096 (max) results to 512 bytes and then base64 result to approx 684 char. So we set to 1000 to be within limits.